Xamarin.Forms.Core/Element.cs

```diff
@@ -125,6 +125,19 @@ public string StyleId
 		[EditorBrowsable(EditorBrowsableState.Never)]
 		public ReadOnlyCollection<Element> LogicalChildren => LogicalChildrenInternal;
 
+		[EditorBrowsable(EditorBrowsableState.Never)]
+		public virtual IEnumerable<T> WalkChildren<T>() where T : Element
```

Comment:
Is there a reason this needs to be depth-first? If not, `Element` already has a method `Descendants()` which iterates breadth-first over its children.

Reply:
Oh. This method I did not notice. `Descendants()` will be enough. Thanks!